ckanext/security/fanstatic/login_ajax.js

```diff
@@ -27,13 +27,14 @@
 
   var validateLoginCredentials = function (e) {
     e.preventDefault()
+    hideError()
```

Comment:
Minor tidy up here to clear the previous error message when submitting the form.